### PR TITLE
Changes to EventHandler for Exiled ScpsInfoDisplay.

### PR DESCRIPTION
**Pull request by SnivyFilms**
Please note the majority of these changes are purely cosmetic, which is why I want to bring your attention to EventHandler.cs

You had 
.Replace("%arhealth%", target.ArtificialHealth > 0 ? target.ArtificialHealth.ToString() : "")
which was detecting the Artificial Health of SCPs, using Exiled, SCPs doesnt have ArtificialHealth, it has Humeshield.

The change is this.
.Replace("%arhealth%", Math.Floor(target.HumeShield) >= 0 ? Math.Floor(target.HumeShield).ToString() : "")
This will display the Humeshield if the config has %arhealth% and it will not vanish when it is depleted

### CODE REVIEW

File: ScpsInfoDisplay/Config.cs [hunk 1]
```diff
@@ -13,12 +13,12 @@ public class Config : IConfig
         [Description("Display strings. Format: Role, display string.")]
         public Dictionary<RoleTypeId, string> DisplayStrings { get; set; } = new Dictionary<RoleTypeId, string>()
         {
-            { RoleTypeId.Scp049, "<color=#D51D1D>SCP-049 [%healthpercent%% Zombies: %zombies%] %distance%</color>" },
-            { RoleTypeId.Scp079, "<color=#D51D1D>SCP-079 [%generators%%engaging%/3] Lvl: %079level% Exp: %079experience% Energy: %079energy%</color>" },
-            { RoleTypeId.Scp096, "<color=#D51D1D>SCP-096 [%healthpercent%%] %distance%</color>" },
-            { RoleTypeId.Scp106, "<color=#D51D1D>SCP-106 [%healthpercent%%] Vigor: %106vigor%% %distance%</color>" },
-            { RoleTypeId.Scp173, "<color=#D51D1D>SCP-173 [%healthpercent%%] %distance%</color>" },
-            { RoleTypeId.Scp939, "<color=#D51D1D>SCP-939 [%healthpercent%%] %distance%</color>" },
+            { RoleTypeId.Scp049, "<color=#FF0000><size=30>SCP-049</color> [<color=#19FF40>%healthpercent%%</color>]\n<color=#19B2FF>Zombies: %zombies%</color></size>" },
```
Comment (bladuk) [inline ScpsInfoDisplay/Config.cs:16]: This plugin has two versions: for EXILED and NWPluginAPI. If you want to make changes to the config, do it for both versions.

File: ScpsInfoDisplay/ScpsInfoDisplay.cs [hunk 2]
```diff
@@ -8,9 +8,9 @@ internal class ScpsInfoDisplay : Plugin<Config>
     {
         public override string Prefix => "scpsinfodisplay";
         public override string Name => "ScpsInfoDisplay";
-        public override string Author => "bladuk.";
+        public override string Author => "Vicious Vikki";
```
Comment (bladuk) [inline ScpsInfoDisplay/ScpsInfoDisplay.cs:11]: Please do not change the plugin's author until you have made really meaningful changes to the code.

**Review by bladuk — CHANGES_REQUESTED** [review]

File: ScpsInfoDisplay/ScpsInfoDisplay.cs (hunk 2 above)
Comment (SnivyFilms) [reply ScpsInfoDisplay/ScpsInfoDisplay.cs:11]: Please note the majority of these changes are purely cosmetic, which is why I want to bring your attention to EventHandler.cs

File: ScpsInfoDisplay/Config.cs (hunk 1 above)
Comment (SnivyFilms) [reply ScpsInfoDisplay/Config.cs:16]: Please note the majority of these changes are purely cosmetic, which is why I want to bring your attention to EventHandler.cs

**Comment by SnivyFilms** [comment]
Please note the majority of these changes are purely cosmetic, which is why I want to bring your attention to EventHandler.cs

If you cannot read that first line I will just close the request as it is an obvious optical failing

**Comment by bladuk** [comment]
I see. In which case it would be more correct to open an issue instead of PR.
I'll fix the hume shield variable ASAP.

**Pull request closed without merging**